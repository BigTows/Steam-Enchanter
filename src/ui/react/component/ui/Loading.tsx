import React from "react";

export default function Loading() {
  return (
    <div className="market_multi_throbber">
      <div className="LoadingWrapper">
        <div className="LoadingThrobber">
          <div className="Bar Bar1" />
          <div className="Bar Bar2" />
          <div className="Bar Bar3" />
        </div>
      </div>
    </div>
  );
}