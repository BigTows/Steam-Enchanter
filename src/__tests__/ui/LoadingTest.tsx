import renderer from "react-test-renderer";
import React from "react";
import Loading from "../../ui/react/component/ui/Loading";

test("Loading component test", () => {
  let component = renderer.create(
    <Loading />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});