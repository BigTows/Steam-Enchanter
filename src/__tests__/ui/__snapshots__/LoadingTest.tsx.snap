// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component test 1`] = `
<div
  className="market_multi_throbber"
>
  <div
    className="LoadingWrapper"
  >
    <div
      className="LoadingThrobber"
    >
      <div
        className="Bar Bar1"
      />
      <div
        className="Bar Bar2"
      />
      <div
        className="Bar Bar3"
      />
    </div>
  </div>
</div>
`;
